packages/server/src/main/java/software/uncharted/terarium/hmiserver/service/data/SimulationService.java

```diff
@@ -84,4 +93,41 @@ public void copySimulationResultToDataset(final Simulation simulation, final Dat
 			}
 		}
 	}
+
+	@Observed(name = "function_profile")
+	public Map<String, FileExport> exportAssetFiles(final UUID simId, final Schema.Permission hasReadPermission)
```

Comment:
This looks like the money, will it be imported correctly - go into S3 somehow?

Reply:
Yes, I change the format of how we import the `FileExport` class, so that i can specify the fullpath, since result files do into a diff key pattern than filenames.